ui/hooks/useDisplayName.ts

```diff
@@ -42,6 +46,15 @@ export function useDisplayNames(
     const firstPartyContractName = firstPartyContractNames[index];
     const singleContractInfo = contractInfo[index];
     const watchedNftName = watchedNftNames[value.toLowerCase()]?.name;
+    const nftCollectionProperties = nftCollections[value.toLowerCase()];
+
+    let nftCollectionName;
+    let nftCollectionImage;
+
+    if (nftCollectionProperties?.isSpam === false) {
+      nftCollectionName = nftCollectionProperties?.name;
```

Comment:
Minor, but could we define an `isSpam` variable, then assign both of these using `const` and a ternary?

Reply:
Done